spec/factories/attachments.rb

```diff
@@ -1,15 +0,0 @@
-# frozen_string_literal: true
```

Comment:
The attachable attachable does not depend on this factory, because it will find the attachment by hash.

Also since attachment is one to one mapping to file now, factories does not really make sense IMO, we don't want to create files.